### DISCUSSION TITLE
Pinokio install?

### DISCUSSION THREAD

**Discussion by osyahbana** [av/harbor#52] [Q&A]
I have ComfyUI and Open WebUI through Pinokio. Is there anyway to "move" the management of these installs to Harbor?

**Comment by BradKML** [comment]
Imagine if Harbor has coverage-parity similar to Pinokio, would be better. ComfyUI has a plugin manager so transferring stuff should be default easy. Not sure about Open WebUI tho https://github.com/Comfy-Org/ComfyUI-Manager
